src/betr/betr_core/TransportMod.F90

```diff
@@ -1180,6 +1180,8 @@ subroutine backward_advection(zi, us,  dtime, zold, bstatus)
      endif
 
      call DDeallocate(gtype)
+     nullify(gtype)
+     deallocate(Extra_inst)
```

Comment:
`gtype` is an alias pointer to `Extra_inst`, need to nullify it.
`Extra_inst` is allocated locally, need to also deallocate it.